### ISSUE TITLE
No image on the docs

### ISSUE THREAD

**Issue by Rom1deTroyes** [Smoothieware/smoothieware-website-v1#1]
Hi, and happy to see that the Smoothie projects are still on :-)

There is no `/images/` folder, so all the documentation relaying on pictures is hard to follow :-(

By the way, some links are broken, making navigation hazardous in this great knowledge base...

What is the best way I can help to give some cares to this documentation ?

Cheers,
    Romain

**Comment by Rom1deTroyes** [comment]
About the links, I feel there are two kinds of information on the V1 site : specific part for the v1 board (configuration, wiring, firmware, tool chains, ...), more general topics about CNC or GCodes, and things between that may be useful for v2 (or not...) like panel accessories or CAM software.
Some pages could obviously be landing pages and should stay in the smoothieware.org/PAGE like [getting-smoothie](http://smoothieware.org/getting-smoothie)

I don't know what is on the way for the V2 website, but if you have thoughts about landing pages you want to keep, just list it :-)

**Comment by arthurwolf** [comment]
Thanks a lot for the issue (hope you're doing alright).

I added the images ( https://github.com/Smoothieware/smoothieware-website-v1/tree/main/docs/images ) but I think they're not in the right location. I'll try to figure that out when I find the time.

About the broken links, I need to take the time to go over each page and rewrite all of them (the old site relied on a url rewriting system that's not available on github, breaking many of the links). Anyone willing to fix any link (or knowing a solution to not to have to do it manually) it's very welcome.

About v2, it'll be a completely different repo and website when we implement it, so we don't need to worry about it for this repo.

**Comment by Rom1deTroyes** [comment]
Nice, at least, they are not lost :-)

I'm working on our well known CNC, but I have to refresh all my knowledge about it, and especially the wiring part (I think the config will work as it, hopefully).

What I can do, as I will read a lot of pages, is moving all links I see to local relative files in markdown format, and do the same thing with the images... This can be a base for a more systematic conversion :-)
